BlueEastCode/bluerain-plugin-redux
# Feature/render prop pattern

## What I did
Converted HOC according to RenderPropPattern
## How to test
By adding plugin  in App bluerain.js
Is this testable with jest or storyshots?
Test are written
Are any of the existing tests are failing ?
No
Does this need an update to the documentation?
no
Does it have breaking changes ?
no
If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
